### PR TITLE
 Add class person to develop

### PR DESCRIPTION
**Pull request by 7Woker7**
#2  issue solved

### CODE REVIEW

File: Person/Program.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            string a = "Hello world!";
+            Console.WriteLine(a);
+        }
+    }
```
Comment (arsenbsrb) [inline Person/Program.cs:13]: You must edit title for class _Program_, this class should be with title _Person_. 
Delete variable _a_ then add integer variable _b_ with value 55. 
@7Woker7

File: ConsoleApp/Program.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = 1, 
+                b = 2;
```
Comment (7Woker7) [inline ConsoleApp/Program.cs:10]: @arsenbsrb  you should make these two variables, been provided by the user #2

Comment (arsenbsrb) [reply ConsoleApp/Program.cs:10]: Ok

File: Person/Program.cs (hunk 1 above)
Comment (7Woker7) [reply Person/Program.cs:13]: _I don't care_. **Do it yourself** @arsenbsrb

**Pull request merged**